atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -309,14 +309,8 @@ private void registerMetrics(GenericObjectPool<CassandraClient> pool) {
         registerPoolMetric("meanBorrowWaitTimeMillis", pool::getMeanBorrowWaitTimeMillis);
```

Comment:
I'm curious about this metric as well, I thought we don't wait at all, but waiting is done in the CassandraRetrier. So in effect this might be timing a memory operation of checking out a connection from the pool

Reply:
Hmm. There are places where this does run into 40+ ms, though I'd be fine with removing it: not sure what this would be used for, and you only have one data point in a long while. So while I don't think it's a memory op it doesn't seem very valuable